src/main/java/com/coinbase/exchange/api/withdrawals/WithdrawalsService.java

```diff
@@ -1,49 +1,53 @@
 package com.coinbase.exchange.api.withdrawals;
 
-import com.coinbase.exchange.api.entity.*;
+import com.coinbase.exchange.api.entity.CoinbasePaymentRequest;
+import com.coinbase.exchange.api.entity.CryptoPaymentRequest;
+import com.coinbase.exchange.api.entity.MonetaryRequest;
+import com.coinbase.exchange.api.entity.PaymentRequest;
+import com.coinbase.exchange.api.entity.PaymentResponse;
 import com.coinbase.exchange.api.exchange.GdaxExchange;
+import java.math.BigDecimal;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.ParameterizedTypeReference;
 import org.springframework.stereotype.Component;
 
-import java.math.BigDecimal;
-
 /**
  * Created by robevansuk on 16/02/2017.
  */
 @Component
 public class WithdrawalsService {
 
-    private static final String WITHDRAWALS_ENDPOINT = "/withdrawals";
-    private static final String PAYMENT_METHOD = "/payment-method";
-    private static final String COINBASE = "/coinbase-account";
-    private static final String CRYPTO = "/crypto";
+  private static final String WITHDRAWALS_ENDPOINT = "/withdrawals";
+  private static final String PAYMENT_METHOD = "/payment-method";
+  private static final String COINBASE = "/coinbase-account";
+  private static final String CRYPTO = "/crypto";
 
 
-    @Autowired
-    GdaxExchange gdaxExchange;
+  @Autowired
+  GdaxExchange gdaxExchange;
 
-    public PaymentResponse makeWithdrawalToPaymentMethod(BigDecimal amount, String currency, String paymentMethodId) {
-        PaymentRequest request = new PaymentRequest(amount, currency, paymentMethodId);
-        return makeWithdrawal(request, PAYMENT_METHOD);
-    }
+  public PaymentResponse makeWithdrawalToPaymentMethod(BigDecimal amount, String currency, String paymentMethodId) {
+    PaymentRequest request = new PaymentRequest(amount, currency, paymentMethodId);
+    return makeWithdrawal(request, PAYMENT_METHOD);
+  }
 
-    // TODO untested - needs coinbase account ID to work.
-    public PaymentResponse makeWithdrawalToCoinbase(BigDecimal amount, String currency, String coinbaseAccountId) {
-        CoinbasePaymentRequest request = new CoinbasePaymentRequest(amount, currency, coinbaseAccountId);
-        return makeWithdrawal(request, COINBASE);
-    }
+  // TODO untested - needs coinbase account ID to work.
+  public PaymentResponse makeWithdrawalToCoinbase(BigDecimal amount, String currency, String coinbaseAccountId) {
+    CoinbasePaymentRequest request = new CoinbasePaymentRequest(amount.setScale(8, BigDecimal.ROUND_DOWN), currency, coinbaseAccountId);
```

Comment:
This was important.

Reply:
keep this - the rest can be reverted. Also any reason for ROUND_DOWN? Everywhere else in the application ROUND_UP is used.

Reply:
I played around with these when I was using the lib. ROUND_DOWN is used because we don't want to ROUND_UP and not have enough fiat currency to cover the order, which would trigger a failure.